docs/components/Countdown/Web.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Countdown } from "@jobber/components/Countdown";
+import { Button } from "@jobber/components/Button";
+import { Emphasis } from "@jobber/components/Emphasis";
+import { Content } from "@jobber/components/Content";
+import { Heading } from "@jobber/components/Heading";
+import { Text } from "@jobber/components/Text";
+
+export default {
+  title: "Components/Utilities/Countdown/Web",
+  component: Countdown,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof Countdown>;
+
+const BasicTemplate: ComponentStory<typeof Countdown> = args => (
+  <Countdown {...args} />
+);
+
+export const Basic = BasicTemplate.bind({});
+Basic.args = {
+  date: new Date(new Date().getTime() + 25 * 3600 * 1000),
```

Comment:
Two things here:

 - `date={{}}` in code tab, can we fix that but creating a const in the template instead so that it shows in the code example?  
 - When you switch to the code tab, those numbers become NaN

![Screenshot 2023-07-31 at 4 32 14 PM](https://github.com/GetJobber/atlantis/assets/104797704/1da51691-67fe-4780-a77a-fae195fd53e5)

Reply:
Same thing happens in Styling example

![Screenshot 2023-07-31 at 4 33 57 PM](https://github.com/GetJobber/atlantis/assets/104797704/66384081-7c24-442b-be98-5e781aaee4f8)

Reply:
Oh god, it didn't happen on my test, I'm glad you saw it!